hack/dockerfile/install/containerd.installer

```diff
@@ -15,7 +15,7 @@ set -e
 # the binary version you may also need to update the vendor version to pick up
 # bug fixes or new APIs, however, usually the Go packages are built from a
 # commit from the master branch.
-: "${CONTAINERD_VERSION:=v1.5.7}"
+: "${CONTAINERD_VERSION:=v1.5.8}"
```

Comment:
whoops; I forgot to update the Windows Dockerfile accordingly 🙈 opened a follow-up https://github.com/moby/moby/pull/43032